.github/workflows/windows.yml

```diff
@@ -399,7 +400,7 @@ jobs:
             $PIP_PKG_NAME=(Get-ChildItem open3d*-$py_tag-*.whl).Name
           }
           echo "Installing Open3D wheel $PIP_PKG_NAME in virtual environment..."
-          python -m pip install "$PIP_PKG_NAME"
+          python -m pip install "$PIP_PKG_NAME[standard]"
```

Comment:
I feel like `[all], [full], [complete]`, or even `[gui]` might be more informative. Standard gives me the impression that it is a standard minimal set. But this is possibly subjective. @ssheorey what's your opinion?

Reply:
As an FYI, `standard` is used in another popular Python package:

- https://www.uvicorn.org/#quickstart

Others use `all`:

- https://github.com/tiangolo/fastapi/blob/a3f1689d78abef3d5d54ba655ec87cef81bf7384/pyproject.toml#L56